study.md

```diff
@@ -114,15 +124,15 @@ Ruby:
 ```ruby
 array = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
 
-# your answer here
+array.reduce{|a,b| a*b}
 ```
 
 JavaScript:
 
 ```javascript
 const array = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
 
-// your answer here
+array.recuce(x,y=> x*y)
```

Comment:
Typo: `recuce`.  You need to wrap the callback function's parameters in parentheses if there's more than one.